File: app/assets/javascripts/models/meeting_visit.js
_.constructor("MeetingVisit", Model.Record, {
  constructorInitialize: function() {
    this.columns({
      userId: 'key',
      meetingId: 'key',
      updatedAt: 'datetime'
    });

    this.belongsTo('meeting');
    this.belongsTo('user');
  }
});

File: app/models/meeting_visit.rb
class MeetingVisit < Prequel::Record
  column :id, :integer
  column :meeting_id, :integer
  column :user_id, :integer
  column :created_at, :datetime
  column :updated_at, :datetime

  belongs_to :meeting
  belongs_to :user

  def team_ids
    meeting ? meeting.team_ids : []
  end
end
